action.yml

```diff
@@ -8,6 +8,10 @@ inputs:
   release_name:
     description: 'The name of the release. For example, `Release v1.0.1`'
     required: true
+  body:
+    description: 'Text describing the contents of the tag.'
+    required: false
+    default: ''
```

Comment:
I thought it would be null, but you are right. Testing in the node repl in the action runner I saw this:

```javascript
> core.getInput("i know this does not exist")
''
```

I am not typescript savvy, but sure looks like the [`getInput` function guarantees it will return a string](https://github.com/actions/toolkit/blob/master/packages/core/src/core.ts#L60-L66). So I can remove this default.